packages/gatsby-transformer-sharp/src/types.ts

```diff
@@ -34,7 +39,7 @@ export const ImagePlaceholderType = new GraphQLEnumType({
   values: {
     DOMINANT_COLOR: { value: `dominantColor` },
     TRACED_SVG: { value: `tracedSVG` },
-    BASE64: { value: `base64` },
+    BLURRED: { value: `blurred` },
```

Comment:
Isn't this a breaking change?

Reply:
This is a new resolver, so no. It's only alpha users who will be using it